### PR TITLE
Add support for control of Amcrest cameras

### PR DESCRIPTION
**Pull request by cnighswonger**
See Issue #2525 for a full description of the feature added in this pull request.

The company I work for was completely disappointed by the (lack of) Amcrest support. So I have no plans to develop this control type further. We are moving over to Dahua (which Amcrest seems to use as their source), and I plan to introduce a control type for their API in a few weeks.

That said, it is apparent from the forum discussions that an Amcrest control type is desirable, hence this pull request.

Thanks!

### CODE REVIEW

File: scripts/ZoneMinder/lib/ZoneMinder/Control/Amcrest_HTTP.pm [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+    my $self = shift;
+    Debug( "Move Stop/Center" );
+    $self->sendCmd( 'cgi-bin/ptz.cgi?action=start&code=PositionABS&channel=0&arg1=0&arg2=0&arg3=0&arg4=1' );
+}
+
+# Move Camera to Home Position
+# The current API does not support a Home per se, so we'll just send the camera to preset #1
+# NOTE: It goes without saying that the user must have set up preset #1 for this to work.
+
+#sub presetHome
```
Comment (connortechnology) [inline scripts/ZoneMinder/lib/ZoneMinder/Control/Amcrest_HTTP.pm]: This appears to be a misplaced comment

Comment (cnighswonger) [reply scripts/ZoneMinder/lib/ZoneMinder/Control/Amcrest_HTTP.pm]: Fixed.

Thanks!

**Comment by alabamatoy** [comment]
ZM version 1.32.3 seems to have no Amcrest control support.  Is this intentional?

**Comment by connortechnology** [comment]
1.32.3 is old.  This PR was merged AFTER 1.32.3 was released.  So yes, intentional.

**Comment by alabamatoy** [comment]
Ok, so school me.  I just looked at the releases in this git, and it's the latest one I see.  It's also the latest version with a discussion in the forums.  And when I look for updates with apt-get, there are none.  ????

**Comment by connortechnology** [comment]
we havn't had a formal stable release since 1.32.3.  However if you look at the commits and current master branch you will see that development has continued. I am hoping to get 1.34 out in the next few days or at least before Christmas.

To run our current code add the master snapshot ppa.
sudo add-apt-repository ppa:iconnor/zoneminder-master

You will get 1.33.15

**Pull request merged**